doc/source/advanced.rst

```diff
@@ -427,11 +427,6 @@ To get information about the current available resource capacity of your cluster
 
 Runtime Environments
 -----------------------------------
-
-.. note::
-
-    This API is in beta and may change before becoming stable.
-
```

Comment:
I am not sure if it is non experimental anymore.

Reply:
We did announce it as GA in 1.6 release

Reply:
I think here we really wanted to just use the beta stability annotation, but there's no way to do annotate a single kwarg (in this case, `runtime_env`.). However, if the API is no longer going to change, then we can remove this.  My only concern would be the new pluggability changes, but even for those we aren't proposing changing the API for any existing options like `pip`, `conda`, etc.  @edoakes what do you think, are we OK to remove this?

Reply:
Chatted with @edoakes offline. I'll close the PR for now and we can remove this block after we stabilize the API more